samples/dnn/optical_flow.py

```diff
@@ -64,12 +80,16 @@ def motion_to_color(self, flow):
     parser.add_argument('-input', '-i', required=True, help='Path to input video file. Skip this argument to capture frames from a camera.')
     parser.add_argument('--height', default=320, type=int, help='Input height')
     parser.add_argument('--width', default=448, type=int, help='Input width')
-    parser.add_argument('--proto', '-p', default='FlowNet2_deploy_anysize.prototxt', help='Path to prototxt.')
-    parser.add_argument('--model', '-m', default='FlowNet2_weights.caffemodel', help='Path to caffemodel.')
+    parser.add_argument('--raft', '-r', action='store_true', help='Model to estimate optical flow. (RAFT or FlowNet)')
+    parser.add_argument('--proto', '-p', help='Path to prototxt.')
+    parser.add_argument('--model', '-m', required=True, help='Path to caffemodel.')
```

Comment:
Please make sample work with —model=path_to_model.onnx. readNet can handle import this

Reply:
I didn't seem to understand this one. Should I check this sample by passing in the --model parameter for a .onnx file?

Reply:
Exactly

Reply:
Yes it's working I checked